### ISSUE TITLE
0.5.0 fails to compile.

### ISSUE THREAD

**Issue by ianp** [james-henderson/chord#29]
As soon as I require `[chord.client :refer [ws-ch]]` I get the following error 

```
clojure.lang.ExceptionInfo : Referred var fressian-cljs.fns/lookup does not exist at line 1
file:/Users/ianp/.m2/repository/net/unit8/fressian-cljs/0.1.0/fressian-cljs-0.1.0.jar!/fressian_cljs/reader.cljs
```

Which is annoying as I don't plan on using Fressian. I think that keeping the chord-fressian and chord-transit formats as separate dependencies was a better approach that lumping everything together in one project, this would also make it easier to add additional formats later on.

**Comment by james-henderson** [comment]
Thanks Ian - I've fixed this as of 0.6.0 - binary support is now in a separate namespace so shouldn't impact the require of chord.client.

Having the formats in separate libraries was a pain while the internals of Chord were changing - with the advent of AJAX support I had to make the format support more general, which would then have required co-ordinating changes in both chord-fressian and chord-transit. Users can still add additional formats through the defmulti, but we've moved common formats into the main library for ease of import.

Cheers,

James
